pkg/getter/httpgetter.go

```diff
@@ -111,5 +112,18 @@ func (g *HTTPGetter) httpClient() (*http.Client, error) {
 
 		return client, nil
 	}
+
+	if g.opts.insecureSkipVerifyTLS {
+		client := &http.Client{
+			Transport: &http.Transport{
```

Comment:
I'm wondering if this should be consolidated with the `if` block above. The `http.ProxyFromEnvironment` and other settings should probably apply here as well , for example (see line 109)

Reply:
You're right - I'll come up with another way on passing the variable.

Reply:
I decided to not consolidate the two if clauses since ca cert check skipping has nothing in common with client auth. Instead I just added `http.ProxyFromEnvironment`.